templates/web/thamesmead/header_logo.html

```diff
@@ -0,0 +1,2 @@
+<a href="https://shropshire.gov.uk" id="site-logo">Peabody</a>
```

Comment:
Cobrand name

Reply:
I have update the link, however instead of using Thamesmead, I kept Peabody, because the link is going to take you to the Peabody website. But let me know if you think it should be Thamesmead.